lib/tracker_api/resources/epics_search_result.rb

```diff
@@ -0,0 +1,12 @@
+module TrackerApi
+  module Resources
+    class EpicsSearchResult
+      include Shared::Base
+
+      attribute :epics, Array
```

Comment:
If you specify the type of object in the array Virtus will coerce them for you. `attribute :epics, Array[Resources::Epic]`